broker/src/main/java/io/camunda/zeebe/broker/partitioning/PartitionManagerImpl.java

```diff
@@ -123,64 +127,68 @@ public CompletableFuture<Void> start() {
 
     healthCheckService.registerBootstrapPartitions(memberPartitions);
     for (final var partitionMetadata : memberPartitions) {
-      final var id = partitionMetadata.id().id();
-      partitionStartup
-          .bootstrap(partitionMetadata)
-          .whenComplete(
-              (startedPartition, throwable) -> {
-                if (throwable != null) {
-                  LOGGER.error("Failed to start partition {}", id, throwable);
-                  onHealthChanged(id, HealthStatus.DEAD);
-                } else {
-                  LOGGER.info("Started partition {}", id);
-                  final var zeebePartition = startedPartition.zeebePartition();
-                  final var raftPartition = startedPartition.raftPartition();
-
-                  zeebePartition.addFailureListener(
-                      new PartitionHealthBroadcaster(id, this::onHealthChanged));
-                  diskSpaceUsageMonitor.addDiskUsageListener(zeebePartition);
-                  adminAccess.put(id, zeebePartition.getAdminAccess());
-                  zeebePartitions.add(zeebePartition);
-                  raftPartitions.add(raftPartition);
-                }
-              });
+      bootstrapPartition(partitionMetadata);
```

Comment:
:thought_balloon: Not really a change in this PR. Now, we just starts partitions async and never waits for the partitions start to be completed. It would be nice if we can log once all partitions have started, even if we don't block the startup of `PartitionManager` until then. This will help to debug in case a partition never starts up. Now we have to search through the logs to find if every starting partition has completed their startup.

Reply:
That is sort of covered by the health checks. All bootstrap partitions are registered ahead of time and will stay  unhealthy ("Services not installed") until the partition is actually started.